src/main/java/com/duffel/model/response/SeatMap.java

```diff
@@ -44,4 +49,46 @@ public class SeatMap {
     @JsonProperty("slice_id")
     private String sliceId;
 
+    /**
+     * Basic pretty print of a seat map. Doesn't cover everything, but just a start to help with development.
+     * @return basic printable text seat map
+     */
+    public String prettyPrintSeatMap() {
+        StringBuilder output = new StringBuilder();
+        output.append("Seat map ").append(getId()).append("\n");
+
+        cabins.forEach(cabin -> {
+                    output.append("\tCabin class ").append(cabin.getCabinClass()).append("\n");
+                    cabin.getRows().stream().map(row -> {
+                                List<String> rowData = new ArrayList<>();
+                                row.getSections().forEach(section -> {
+                                            rowData.add(section.getElements().stream().map(element ->
+                                                    {
+                                                        if (ElementType.seat.equals(element.getType())) {
+                                                            return ((SeatElement) element).getDesignator() + (((SeatElement) element).getAvailableServices().isEmpty() ? "❌" : "✅");
```

Comment:
Very very minor thing: I'm not sure these are the most obvious symbols to show whether a seat is empty, I initially read the ❌ as "this seat is unavailable" rather than "this seat is empty".

Reply:
Ah, it is that way round, red cross is "this seat can't be booked", as it has no services you can purchase

Reply:
So green tick is "you can book this seat"

Reply:
Ah! Makes sense, I was reading the `isEmpty()` as if it referred to the seat rather than the list of available services 😓